### PR TITLE
Update supplier find command

### PR DESCRIPTION
**Pull request by weesiau**
Find command now can search by either name OR item. Choose with prefix n/ or i/

Will add more test cases later

### CODE REVIEW

File: src/test/java/seedu/address/logic/parser/FindCommandParserTest.java [hunk 1]
```diff
@@ -25,10 +27,10 @@ public void parse_validArgs_returnsFindCommand() {
         // no leading and trailing whitespaces
         FindCommand expectedFindCommand =
                 new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList("Alice", "Bob")));
-        assertParseSuccess(parser, "Alice Bob", expectedFindCommand);
+        assertParseSuccess(parser, COMMAND_WORD + " n/Alice Bob", expectedFindCommand);
```
Comment (Kang-Quan) [inline src/test/java/seedu/address/logic/parser/FindCommandParserTest.java:30]: Maybe in the future n/Alice Bob can be a static variable to make the test file neater. Similar issue found in line 33.

File: src/main/java/seedu/address/logic/parser/FindCommandParser.java [hunk 2]
```diff
@@ -19,15 +23,25 @@ public class FindCommandParser implements Parser<FindCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindCommand parse(String args) throws ParseException {
+        requireNonNull(args);
```
Comment (Kang-Quan) [inline src/main/java/seedu/address/logic/parser/FindCommandParser.java:26]: Good addition to try to catch null in the argument.

File: src/test/java/seedu/address/logic/parser/FindCommandParserTest.java (hunk 1 above)
Comment (weesiau) [reply src/test/java/seedu/address/logic/parser/FindCommandParserTest.java]: Agree, fixed

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 3]
```diff
@@ -2,9 +2,13 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.function.Predicate;
+
 import seedu.address.commons.core.Messages;
 import seedu.address.model.Model;
+import seedu.address.model.person.ItemContainsKeywordsPredicate;
```
Comment (hauchongtang) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:9]: Nice to see that we check the item inside person as well as their name. Placement of this predicate also makes sense👍

**Review by hauchongtang — APPROVED** [review]
LGTM, the addition of checking for `Item` is probably useful for our target audience. 👍

**Pull request merged**